example/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.lightstep.opentelemetry</groupId>
   <artifactId>lightstep-opentelemetry-example</artifactId>
-  <version>0.2.0</version>
+  <version>0.2.2</version>
```

Comment:
As we are not publishing the example at all, maybe we can leave it with version `0.1.0-SNAPSHOT` forever?

Reply:
done